jobbergate-agent/tests/jobbergate/test_submit.py

```diff
@@ -699,7 +707,10 @@ def _mocked_mark_as_submitted(job_submission_id: int, slurm_job_id: int):
 
     test_mapper = manufacture()
 
-    await submit_pending_jobs()
+    with tweak_settings(CACHE_DIR=tmp_path):
+        cached_submission_4 = tmp_path / "4.slurm_job_id"
+        cached_submission_4.write_text("44")
+        await submit_pending_jobs()
```

Comment:
I guess it's missing a case to test if the cache file is created in case it doesn't exist

Reply:
Thanks for the suggestion. I added some assertions on the cache files to cover this scenario as well. Addressed by [5d46d78](https://github.com/omnivector-solutions/jobbergate/pull/607/commits/5d46d78036c039e69ae44ca33e3bdb8b56e35212).